app/grandchallenge/workstations/templatetags/workstations.py

```diff
@@ -79,3 +80,27 @@ def workstation_query(  # noqa: C901
         )
 
     return urlencode(query)
+
+
+@register.simple_tag()
+def workstation_query(**kwargs):
```

Comment:
This template tag shouldn't be used anywhere else now, right?

Reply:
I can see that it is in `rawimageuploadsession_detail.html` - this should be replaced with a button for the new session control, good to find. 

Also used in `display_set.workstation_url`, but I _think_ that property can be updated to use the query string. That seems only to be used when creating a display set from an algorithm (we can decide if opening the viewer for that new display set is the best idea or not).

Reply:
Ah good one. I tested both of these cases and it works.